portal.go

```diff
@@ -1664,6 +1664,28 @@ func (portal *Portal) HandleMessageRevoke(user *User, info *types.MessageInfo, k
 	return true
 }
 
+func (portal *Portal) deleteForMe(user *User, content *events.DeleteForMe) bool {
+	matrixUsers, err := portal.GetMatrixUsers()
+	if err != nil {
+		portal.log.Errorfln("Unable to get Matrix users for portal, to see if DeleteForMe should be handled or not")
```

Comment:
This should actually log the error (also adjusted the error message slightly to match the usual style)

```suggestion
		portal.log.Errorln("Failed to get Matrix users in portal to see if DeleteForMe should be handled:", err)
```